scripts/integer-tests.sh

```diff
@@ -130,8 +130,8 @@ fi
 # Override test-threads number to avoid Out-of-memory issues on GPU instances
 if [[ "${backend}" == "gpu" ]]; then
     if [[ "${BIG_TESTS_INSTANCE}" == TRUE ]]; then
-        test_threads=5
-        doctest_threads=5
+        test_threads=8
```

Comment:
Have we tested this for all configurations? I remember having an out-of-memory exception when tried to run with more than 6 threads some time ago.

Reply:
All tests passed in the CI so it looks good.

Reply:
(I triggered approval tests by applying the `approved` label to the PR before you reviewed)